guides/common/modules/proc_configuring-projectserver-to-use-cross-forest-trust.adoc

```diff
@@ -22,13 +22,14 @@ For example:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-[nss]
-user_attributes=+mail, +sn, +givenname
```

Comment:
Shouldn't this stay the way it was?

Reply:
This is based on Waldirio's case notes in https://issues.redhat.com/browse/SAT-23355. He requests changing the NSS config line to use the attribute mappings configured in [ldap_user_extra_attrs](https://jhrozek.fedorapeople.org/sssd/1.11.90/man/sssd-ldap.5.html) and moving the [nss] section below the line with ldap_user_extra_attrs. It does seem to make sense based on the description in sssd-ldap(5) but I'm happy to change this if you have any concerns @adamruzicka